app/assets/stylesheets/signup.css.scss

```diff
@@ -25,6 +25,11 @@ input[type="text"], input[type="alt_email"], input[type="email"], input[type="pa
   display:none;
 }
 
-#sign_up_button:focusout{
-	background-color:grey;
+#sign_up_button{
+  background-color:grey;
```

Comment:
Interesting to note that "All gray colors are spelled as gray (not grey).  All modern browsers accept both gray and grey, but early versions of Internet Explorer did not recognize grey."
Source: https://www.w3schools.com/colors/colors_shades.asp